qiita_pet/templates/study_base.html

```diff
@@ -0,0 +1,72 @@
+{% extends sitebase.html %}
+{% block head %}
+<script type="text/javascript">
+  function fill_base(sid) {
+  $.get( "/study/description/baseinfo/", { study_id: sid} )
+    .done(function( data ) {
+      $("#study-main").html(data);
+      $("#study-prep-graph").hide();
+      $("#study-file-breadcrumbs").hide();
+    });
+  }
+
+  $(document).ready(function() {
+    fill_base({{study_info['study_id']}});
+  });
+</script>
+{% end %}
+{% block content %}
+<div class="row">
+  <div class="col-md-3">
+  <h3>Study Information</h3>
+    <a href="#">Sample Template</a><br />
+    <a href="#">Sample Summary</a><br />
+    <a href="/study/upload/{{study_info['study_id']}}">Upload Files</a><br />
+    <a href="#">Edit Study Information</a><br />
+    <a href="#">Delete Study</a>
+
+    <h3>Data Types</h3>
+    <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+      <div class="panel panel-default">
+      {% for dt in prep_info %}
+      {% set cleaned_dt = dt.replace(' ', '_') %}
+        <div class="panel-heading" role="tab" id="heading{{cleaned_dt}}">
+          <h4 class="panel-title">
+            <a role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse{{cleaned_dt}}" aria-expanded="true" aria-controls="collapse{{cleaned_dt}}">{{dt}}</a>
+          </h4>
+        </div>
+        {% end %}
+        {% for prep in prep_info[dt] %}
```

Comment:
OK, got it, that makes perfect sense, we should really keep an eye
opened for this as otherwise it will lead to a rather strange-looking
UI.

On (Dec-14-15|12:50), Joshua Shorenstein wrote:
> > +    <a href="/study/upload/{{study_info['study_id']}}">Upload Files</a><br />
> > +    <a href="#">Edit Study Information</a><br />
> > +    <a href="#">Delete Study</a>
> > +
> > +    <h3>Data Types</h3>
> > +    <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
> > +      <div class="panel panel-default">
> > +      {% for dt in prep_info %}
> > +      {% set cleaned_dt = dt.replace(' ', '_') %}
> > +        <div class="panel-heading" role="tab" id="heading{{cleaned_dt}}">
> > +          <h4 class="panel-title">
> > +            <a role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse{{cleaned_dt}}" aria-expanded="true" aria-controls="collapse{{cleaned_dt}}">{{dt}}</a>
> > +          </h4>
> > +        </div>
> > +        {% end %}
> > +        {% for prep in prep_info[dt] %}
> 
> I'm leveraging the API on this one, since this design keeps with the current assumption that the API call object only returns what information is viewable by the user. 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1571/files#r47555626